README.md

```diff
@@ -401,6 +401,25 @@ Button.defaultProps = {
 };
 ```
 
+#### Getting the theme outside styled components
+
+If you ever need to get the current `theme` outside styled components (e.g. inside render functions), you can use the `withTheme` Higher Order Component:
+
+```JS
+import { withTheme } from 'styled-components'
+
+class Component extends React.Component {
+  render() {
+    const { theme } = this.props
+
+    console.log('Current theme: ', theme);
+    // ...
+  }
+}
+
+export default withTheme(Component)
+```
```

Comment:
This can only live inside theming.md. No need for this to be in the README, it's not a core API!